### ISSUE TITLE
Ressurecting Virtious?

### ISSUE THREAD

**Issue by dportuesi** [MinestrapTeam/Minestrappolation-4#101]
I'm in. We need to brainstorm everything we want back and everything we want to change.

EDIT: https://docs.google.com/document/d/1NFPuiuQff56MCJPe1NJAirybY4LmTrfIJC26dELqMzs/edit

**Comment by Delocuro** [comment]
Lol, read that forum post, I take it? I'd be down for that, but I'll be real, we're _probably_ going to have to re-code it all from scratch. If you're up for the challenge though, I can try to put together a short features list for it.

The one thing to note, however, is that we'll probably need to integrate it directly into Minestrapp rather than having it as its own mod. Aside from not having enough time to manage two separate codebases at once, I also had a lot of ideas as to how the Virtious can fit into the Minestrapp tech tree, and integrating those via inter-mod support will be like four times as hard as just coding them in as the same mod.

**Comment by Clashsoft** [comment]
:+1:

**Comment by Delocuro** [comment]
I've actually already started this by coding in some of the blocks and items, which will be obtainable via the Dimensional Fishery until one of us has enough time to actually code in the dimension, world gen, and portal mechanics. We can do the same thing with any of our other past dimensions or mods as well, so if anyone has dimensions they'd like to see supported, feel free to list those here.

**Comment by Clashsoft** [comment]
Do you plan on doing something with the Aerius (or whatever that was called?) I really liked it and it fits the Minestrap theme

**Comment by Clashsoft** [comment]
I would even be willing to help (mostly portal and world gen code as well as advanced mechanics).

**Comment by Delocuro** [comment]
I was actually hoping to at some point as well, if I can find the time -- I even have some vague ideas  formulated as to how it can fit into the existing minestrapp tech tree. If you wanted to figure out how to get the world gen and portals working in 1.8.8 (you can probably just use placeholder blocks like dirt and stone for now since it'll be easy enough to change them later when we add the actual blocks), I can try and better plan out some of the basic features and stuff for that and get them added as soon as I have time to. Even if we don't get the whole thing added before the next update, we can still add most of the blocks to survival mode via the Dimensional Fishery.

**Comment by Clashsoft** [comment]
Sounds like a plan. I can probably re-use the portal and world code from the ancient artifact known as the Clashsoft Library, replacing coordinate and metadata stuff with the BlockPos thingys.

**Comment by Delocuro** [comment]
Alright, well 4.3.0 is pretty close to being released, and we're adding quite a few Virtious blocks and items in that update, so we should be able to start coding the dimension itself pretty much right after we finish updating to 1.9 if you guys are up for it. :D

**Comment by Quizer9O8** [comment]
Isnt the Aerius a Aether look-a-like?
What purpose would this dimension have?

**Comment by werty1124** [comment]
@Quizer9O8 Yea basically, and it will have it's purposes.

**Comment by Quizer9O8** [comment]
I hope it'll be nothing like the Aether (except its in the sky of course) and what would happen if you reached the bottom of this dimension? will you be teleported back to the overworld at a 500 blocks heights?

**Comment by Clashsoft** [comment]
@Quizer9O8 exactly, and you can also reach it by flying above `y 255` in creative mode. What sets it apart from the Aether are its many biomes and a less 'cheaty' dungeon system (they do not reset, and there are more kinds). Other than that, we have quite a lot of items, including new gear and wearables that can be equipped in new inventory slots.

**Comment by Quizer9O8** [comment]
Ok but the dimension i'm really hyped about is the Virtious. I even made some music for it since its an alien like world with crystal like trees and stuff.
